openfisca_uk/parameters/benefits/benefit_cap/london_no_children.yaml

```diff
@@ -0,0 +1,7 @@
+description: Benefit cap for benefit units without children in London
```

Comment:
Low-priority: Is there a more "mapped" way to do this, e.g. a dict depending on the geography? London/non-London is pretty straightforward but I'm wondering if there'll be a cleaner way to introduce UBI supplements in each of the regions.

Reply:
I think [this openfisca feature](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#computing-a-parameter-that-depends-on-a-variable-fancy-indexing) is probably the best way to do it, I agree about the inefficiency of the current structure